{"name": "test_validlogin", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_scripts.test_validlogin.TestValidLogin object at 0x00000277F1373C70>\n\n    @pytest.mark.smoke\n    @pytest.mark.run(order=1)\n    def test_validlogin(self):\n        userName = Excel.get_cellValue(\"data/Book3.xlsx\",\"ValidLogin\",2,1)\n        passWord = Excel.get_cellValue(\"data/Book3.xlsx\", \"ValidLogin\", 2, 2)\n        loginpage = LoginPage(self.driver)\n        loginpage.set_username(userName)\n        loginpage.set_password(passWord)\n        loginpage.click_loginbutton()\n        time.sleep(5)\n        homepage = HomePage(self.driver)\n        result = homepage.verify_homepage_is_display(self.wait)\n        if result == False:\n            assert True\n        else:\n            allure.attach(self.driver.get_screenshot_as_png(),name=\"validlogin\",attachment_type=AttachmentType.PNG)\n>           assert False\nE           assert False\n\ntest_scripts\\test_validlogin.py:30: AssertionError"}, "attachments": [{"name": "validlogin", "source": "a3069bee-4bb7-42b2-ba48-0439b027e2b7-attachment.png", "type": "image/png"}], "start": 1668416834087, "stop": 1668416839401, "uuid": "34d9bd6a-f601-49ac-b94d-aca16cfff2ab", "historyId": "32234e812278dd24ef7842b59051daea", "testCaseId": "401c0ffc0022c0dedcf5e6617b593ffb", "fullName": "test_scripts.test_validlogin.TestValidLogin#test_validlogin", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "test_scripts"}, {"name": "suite", "value": "test_validlogin"}, {"name": "subSuite", "value": "TestValidLogin"}, {"name": "host", "value": "SRCHM"}, {"name": "thread", "value": "15980-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_scripts.test_validlogin"}]}